app/controllers/admin/api/metrics_base_controller.rb

```diff
@@ -1,24 +1,13 @@
 class Admin::Api::MetricsBaseController < Admin::Api::ServiceBaseController
+  include MetricParams
 
   protected
 
-    def metric
-      raise NotImplementedError.new "#{self.class.name} should implemented metric method"
-    end
+  def metric
+    raise NotImplementedError.new "#{self.class.name} should implemented metric method"
+  end
 
-    def metrics
-      @metrics ||= service.metrics
-    end
-
-    def metric_params
-      parameters = params.fetch(:metric)
-
-      if parameters[:name].present?
-        # as of February 2014, notify that this provider is using a deprecated api
-        deprecated_api "`name' parameter for metric/method api was deprecated, `system_name' should be used for provider #{current_account.id}/#{current_account.org_name}"
```

Comment:
This is a breaking change. OK to proceed?

cc @3scale/product  @3scale/support @3scale/documentation

Reply:
@guicassolato Do we need to document this? If so, we need a Doc JIRA.

Reply:
@porueesq, it's just a heads-up now. I don't know if it needs documentation. Probably the other way around I think, i.e. if we currently have something documented about this (I don't think so), then we need to remove it. Otherwise, it's just "release notes".